source/MaterialXRenderGlsl/TextureBaker.cpp

```diff
@@ -444,10 +440,14 @@ BakedDocumentVec TextureBaker::createBakeDocuments(DocumentPtr doc, const FileSe
 
     DefaultColorManagementSystemPtr cms = DefaultColorManagementSystem::create(genContext.getShaderGenerator().getTarget());
     cms->loadLibrary(doc);
-    if (_codeSearchPath.isEmpty())
-        genContext.registerSourceCodeSearchPath(getDefaultSearchPath());
-    else
+    if (!_codeSearchPath.isEmpty())
```

Comment:
@ashwinbhat , @nicolassavva-autodesk : keeping this code alive. Do we still need this?

Reply:
getDefaultSearchPath() works only for MtlxView I think

Reply:
Yes. The API call was removed and the code migrated to be inside MTLXView only. The checking for 
path / "libraries" was added but it's probably no better. Best to set an explicit path, thus I've left the API for this in while it's been removed from LucasFilm main.